### DISCUSSION TITLE
Injecting a custom Toast API / changing toast duration

### DISCUSSION THREAD

**Discussion by sannajammeh** [payloadcms/payload#1101] [Roadmap]
I absolutely love Payload so far. The only thing I don't like however is the large and long toast messages. Is there any way to completely hide them and/or make them smaller & reduce their duration?

**Comment by jmikrut** [comment]
Good call. I think I agree with you. One note, you can override their size and / or completely hide them pretty easily with custom CSS, but I think that the team and I should take a crack at revising their aesthetic. So I'll tag this as `planned`. 👍

**Reply by jmikrut** [reply]
Also - SUPER pumped to hear that you are loving Payload so far! Welcome!

**Comment by denolfe** [comment]
It was mentioned in Discord that some editors think the progress bar is actually _showing the save progress_. It might be good for us to think about this aspect at the same time.

**Comment by rktyt** [comment]
In our case, the error notifications disappear too quickly, and we sometimes miss them. Therefore, we want to prevent them from disappearing automatically.

Still doesn't seem to be easily achievable in the current version (v3.43.0).
